doc/manuals/admin/index.md

```diff
@@ -0,0 +1,67 @@
+# Installing
+
+At the moment, the only supported way of using Quantumleap is through Docker, so there's no actual "installation" of Quantumleap.
```

Comment:
At the moment, the only actively supported distribution of QuantumLeap is Docker. Though, you can build and install it from sources, but no guidance is provided for such installation at the moment.